tests/ui/pages/discopane.py

```diff
@@ -0,0 +1,44 @@
+from pypom import Page
+from selenium.webdriver.common.by import By
+from selenium import webdriver
+import pytest
+
+
+class DiscoveryPane(Page):
+
+    _discopane_content_locator = (By.CSS_SELECTOR, '.disco-pane')
+    _play_video_locator = (By.CSS_SELECTOR, '.play-video')
+    _uninstalled_toggles = (By.CSS_SELECTOR, '.switch.uninstalled')
+    _close_video_link = (By.CSS_SELECTOR, '.close-video')
+    _see_more_addons_locator = (By.CSS_SELECTOR, '.amo-link')
+    _search_box_locator = (By.ID, 'lst-ib')
+
+    def open_addons(self):
+        elem = self.find_element(*self._search_box_locator)
+        elem.send_keys(Keys.COMMAND + Keys.SHIFT + 'a')
+
+    def play_video(self):
+        self.find_element(*self._play_video_locator).click()
```

Comment:
My instincts tell me we should wait for the video to be playing. That way we avoid failing if the video takes a moment to register as playing, and simply timeout if it doesn't start playing.